cmd/get/launch_plan.go

```diff
@@ -0,0 +1,59 @@
+package get
+
+import (
+	"context"
+	"github.com/golang/protobuf/proto"
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flytectl/pkg/adminutils"
+	"github.com/lyft/flytectl/pkg/printer"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flytestdlib/logger"
+)
+
+var launchplanColumns = []printer.Column{
+	{"Version", "$.id.version"},
+	{"Name", "$.id.name"},
+	{"Type", "$.closure.compiledTask.template.type"},
```

Comment:
can we include launch plan [state](https://github.com/lyft/flyteidl/blob/master/protos/flyteidl/admin/launch_plan.proto#L106) and [schedules](https://github.com/lyft/flyteidl/blob/master/protos/flyteidl/admin/launch_plan.proto#L125) too? this way we don't lose information we have in flyte-cli and it's really useful to get an overview of what's scheduled when listing launch plans